x/emissions/proto/emissions/v9/query.proto

```diff
@@ -555,6 +540,18 @@ service QueryService {
     option (cosmos.query.v1.module_query_safe) = true;
     option (google.api.http).get = "/emissions/v9/latest_forecaster_weight/{topic_id}/{actor_id}";
   }
+
+  // Get worker submission window status and timing
+  rpc GetWorkerSubmissionWindowStatus(GetWorkerSubmissionWindowStatusRequest) returns (GetWorkerSubmissionWindowStatusResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
+    option (google.api.http).get = "/emissions/v9/worker_submission_window_status/{topic_id}";
+  }
+
+  // Get reputer submission window status and timing
```

Comment:
We can have multiple nonce open at the same time so I think we should query the window status for a nonce, not only a topic.

Also maybe we could have a single query to provide both worker and reputer window information